apps/web/src/components/avatar-plus.tsx

```diff
@@ -1,103 +1,61 @@
 'use client';
-
-import { generateAvatarUrl, getInitials } from '@/src/lib/utils';
-import {
-  Avatar,
-  Badge,
-  Popover,
-  Tooltip,
-  type AvatarProps
-} from '@radix-ui/themes';
 import { type TypeId } from '@u22n/utils/typeid';
+import {
+  HoverCard,
+  HoverCardContent,
+  HoverCardTrigger
+} from './shadcn-ui/hover-card';
+import { Avatar, type AvatarProps } from './avatar';
 
 type AvatarPlusProps = {
   size: AvatarProps['size'];
-  imageSize:
-    | '3xs'
-    | '2xs'
-    | 'xs'
-    | 'sm'
-    | 'md'
-    | 'lg'
-    | 'xl'
-    | '2xl'
-    | '3xl'
-    | '4xl'
-    | '5xl';
   users: {
     avatarProfilePublicId: TypeId<'orgMemberProfile' | 'teams' | 'contacts'>;
     avatarTimestamp: Date | null;
     name: string;
+    color?: AvatarProps['color'];
   }[];
 };
 
-export default function AvatarPlus({
-  imageSize,
-  size,
-  users
-}: AvatarPlusProps) {
+export default function AvatarPlus({ size, users }: AvatarPlusProps) {
   const [primary, ...rest] = users;
   if (!primary) {
     return null;
   }
   return (
-    <div className="relative h-fit w-fit">
-      <Tooltip
-        content={primary.name}
-        side="bottom">
-        <button>
-          <Avatar
-            size={size}
-            fallback={getInitials(primary.name)}
-            src={
-              generateAvatarUrl({
-                publicId: primary.avatarProfilePublicId,
-                avatarTimestamp: primary.avatarTimestamp,
-                size: imageSize
-              }) ?? undefined
-            }
-            radius="full"
-          />
-        </button>
-      </Tooltip>
-      <Popover.Root>
-        <Popover.Trigger>
-          <button>
-            <Badge
-              variant="solid"
-              color="gray"
-              radius="full"
-              className="absolute bottom-[-6px] right-[-4px] font-bold">
-              {`+${rest.length}`}
-            </Badge>
-          </button>
-        </Popover.Trigger>
-        <Popover.Content>
-          <div className="flex gap-2 p-2">
+    <div className="relative h-fit w-fit scale-100 overflow-visible">
+      <HoverCard>
+        <HoverCardTrigger>
```

Comment:
`HoverCardTrigger` is a anchor tag, using it inside other anchor tags will cause hydration errors. Add `asChild` attribute to it prevent that issue. we already have one hyrdation error in this PR